scripts/_check_docs_utils.py

```diff
@@ -0,0 +1,726 @@
+# coding: utf-8
+
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Since this doesn't appear to be code that we wrote you should instead write that the code was modified from the pylint source and link to it.

Reply:
Modified it as discussed. Thanks!